schema/build-config-schema.js

```diff
@@ -107,6 +107,11 @@ rules.forEach(function forRule(rule) {
           "description": "Whether to indent the first level of the list",
           "type": "boolean",
           "default": false
+        },
+        "start_indent": {
```

Comment:
You may have realized - the correct pattern is to edit this file, then run the generation script (part of the `ci` script) to update the other 3 files with this content.